src/index.js

```diff
@@ -140,11 +140,12 @@ app.post("/kuskis/:name/shirt", async (req, res) => {
               fs.readFile(shirt_tga, (err, shirtData) => {
                 if (!err) {
                   fs.readFile(shirt_crc, (err, shirtCrc) => {
+                    shirtCrcLE = shirtCrc.readInt32LE(0);
```

Comment:
Here you are missing the `const` keyword

Reply:
Not rly

Reply:
yes rly